Cargo.toml

```diff
@@ -1,13 +1,19 @@
 [package]
 name = "rs-conllu"
 description = "A parser for the CoNLL-U format of the Universal Dependencies project."
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
No need to change, but FYI: I would probably increase the minor version after merging since there have been some other, breaking changes in the meantime on the main branch (renaming `dep` to `deps` for example). I would compile those when creating the new version.

Reply:
Thank you for information! I'll be waiting for new changes!)